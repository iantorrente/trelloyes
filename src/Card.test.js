import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import CardClass from './Card.js';

describe('Test suite for CardClass component', () => {
  it('renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(<CardClass />, div);
    ReactDOM.unmountComponentAtNode(div);
  });
  
  it('renders the UI as expected', () => {
    const tree = renderer
      .create(<CardClass title='card-test' content={['a', 'b', 'c']} />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});