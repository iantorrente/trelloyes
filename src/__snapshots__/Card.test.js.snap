// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test suite for CardClass component renders the UI as expected 1`] = `
<div
  className="Card"
>
  <h3>
    card-test
  </h3>
  <p>
    a
    b
    c
  </p>
</div>
`;
